pkg/drivers/hyperkit/driver.go

```diff
@@ -232,6 +250,56 @@ func (d *Driver) extractKernel(isoPath string) error {
 	return nil
 }
 
+func (d *Driver) setupNFSShare() error {
+	user, err := user.Current()
+	if err != nil {
+		return err
+	}
+
+	hostIP, err := GetNetAddr()
+	if err != nil {
+		return err
+	}
+
+	mountCommands := fmt.Sprintf("#/bin/bash\\n")
```

Comment:
We probably want a "set -e" in there, otherwise it'll continue in the case of an error.